system/usbmsc/Kconfig

```diff
@@ -17,13 +17,20 @@ menuconfig SYSTEM_USBMSC
 if SYSTEM_USBMSC
 
 config SYSTEM_USBMSC_NLUNS
-	int "Number of LUNs"
+	int "Number of const LUNs"
 	default 1
 	---help---
 		Defines the number of logical units (LUNs) exported by the USB
 		storage driver.  Each LUN corresponds to one exported block driver
 		(or partition of a block driver).  May be 1, 2, or 3.  Default is 1.
 
+config SYSTEM_USBMSC_EXT_NLUNS
+	int "Number of max extra LUNs"
+	default 8
+	---help---
+		Defines the max number of logical units (LUNs) exported by the USB
+		storage driver at runtime.
+
 config SYSTEM_USBMSC_DEVMINOR1
 	int "LUN1 Minor Device Number"
 	default 0
```

Comment:
why not use -1 to disable it instead through `-o nc`

Reply:
> why not use -1 to disable it instead through `-o nc`

We can both disable / enable it at runtime.
Set to `0` for disabling at compile time currently.

Reply:
Ok